### ISSUE TITLE
G3000 and GX

### ISSUE THREAD

**Issue by Flix5696** [Working-Title-MSFS-Mods/fspackages#914]
Without latest GX mod, blank waypoints in the G3000 latest beta seem to not appear, and going direct to doesnt freeze MFD

**Comment by Lysdestic** [comment]
Can confirm this behaviour. I've mentioned it a couple of times on discord and on another open issue here on Github but a few weeks back I removed mods and added them back until I found the culprit, and it is NOT navigraph data that causes blank waypoints, but some kind of conflict with workingtitle-gx. Once workingtitle-gx is removed, there are no longer any issues with blank waypoints.

I have no issue using direct to as well. I have had many flights over the past 2-3 weeks in the TBM where I can comfortably say that a conflict with the gx mod is indeed the issue. Thanks for confirming this, Flix!

**Comment by Hornet317CZ** [comment]
I also experienced the described behavior. After removing the GX mod the issue with the G3000 disappeared.

**Comment by cywang90** [comment]
Can somebody who was having this issue please post specific steps to reproduce either the blank waypoints or direct to freeze (ideally with information on specific waypoints/flight plans)? I'm trying to debug the root cause of the incompatibility and having a reliable repro will speed up the process greatly.

**Comment by Flix5696** [comment]
It's been a while, but the random waypoints being blank I would only ever get loading in the flight plan manually. Loading in through the game menu would not give me these. I can try reproduce it when I have some time.

**Comment by Lysdestic** [comment]
> Can somebody who was having this issue please post specific steps to reproduce either the blank waypoints or direct to freeze (ideally with information on specific waypoints/flight plans)? I'm trying to debug the root cause of the incompatibility and having a reliable repro will speed up the process greatly.

Sorry for the delay on this.

Unfortunately, despite being able to consistently recreate this issue for several weeks during December - mid Feb, I am unable to do so now. What I can say is that I almost always test things using one of the two following flight plans:

1) KSEA/16L SEA BUWZO KRATR2.BUWZO KPDX/28L

2) KPDX/28L COUGA KRIEG HAWKZ7.KRIEG KSEA/16L

The method I always fly is to pick a parking spot at my chosen airport, load into the world, then manually program the flight plan into the Garmin by first setting origin and destination, then adding the waypoints and procedures. Usually, once adding the arrival procedure, the waypoints for the arrival would be blank in the flight plan screen, and through trial and error, I tracked it down to the GX as removing it consistently resolved the issue. I tested this enough and posted about it on discord that I felt confident that my findings were correct, but for whatever reason cannot recreate them now.

**Comment by cywang90** [comment]
Out of curiosity with your most recent unsuccessful attempts to repro are you using 0.4.0 or 0.4.1?

**Comment by Lysdestic** [comment]
Most recent, but I switched back to 4.0 (where I know I had encountered it) and still couldn't repro.

It occurs to me that I didn't attempt any approaches, just arrivals...I need to play around a little bit more to see what I'm skipping on reproducing it.

**Comment by Hesynergy** [comment]
To us wannabe coders, what is a push, repo,the process of shared coding on Github?

Thanks & be & help your neighbors(Personal space, and next door),... safe!o

￼Chas

**Comment by Lysdestic** [comment]
I just wanted to add an update to this to say that I think it's probably safe to mark this as a can't reproduce at this point. 

I've had gx, g1000, and g3000 installed and coexisting together for several weeks now and haven't seen this issue pop up at all. I also use navigraph data and have seen no issues with that either. The ONLY time I've managed to see it recently was during a troubleshooting session where I had removed working title mods and still had navigraph...Perhaps something that changed post 4.0 negated the issue or it was just a weird unrelated conflict. I don't think anyone has reported this issue on discord in a while as well.
